app/views/layouts/mailer.html.erb

```diff
@@ -25,6 +25,49 @@
         font-weight: normal;
         margin-bottom: 2.5rem;
       }
+      .email-contents button {
+          background-color: #121111;
+          border-bottom: #121111 2px solid;
+          border-bottom-left-radius: 10px;
+          border-bottom-right-radius: 10px;
+          border-image-outset: 0;
+          border-image-repeat: stretch;
+          border-image-slice: 100%;
+          border-image-source: none;
+          border-width: 2px;
+          border-image-width: 1;
+          border-left-color: #121111;
+          border-left-style: solid;
+          border-right-color: #121111;
+          border-right-style: solid;
+          border-top-color: #121111;
+          border-top-left-radius: 10px;
+          border-top-right-radius: 10px;
+          border-top-style: solid;
+          box-shadow: rgb(3, 78, 70) 0px 2px 0px 0px;
+          box-sizing: border-box;
+          cursor: pointer;
+          font-size: 19px;
+          font-weight: 700;
+          height: 64px;
+          line-height: 25px;
+          padding-bottom: 17.5px;
```

Comment:
could we condense these paddings into just one line like so? (top/bottom left/right)
```suggestion
          padding: 17.5px 25px;
```